test-network-function/accesscontrol/suite.go

```diff
@@ -189,30 +187,32 @@ func runTestOnPods(env *config.TestEnvironment, testCmd testcases.BaseTestCase,
 }
 
 func getCrsNamespaces(crdName, crdKind string, context *interactive.Context) (map[string]string, error) {
-	const expectedNumFields = 2
-	const crNameFieldIdx = 0
-	const namespaceFieldIdx = 0
-
 	gomega.Expect(crdKind).NotTo(gomega.BeEmpty())
 	getCrNamespaceCommand := fmt.Sprintf(ocGetCrNamespaceFormat, crdKind)
 	cmdOut := utils.ExecuteCommandAndValidate(getCrNamespaceCommand, common.DefaultTimeout, context, func() {
 		common.TcClaimLogPrintf("CRD %s: Failed to get CRs (kind=%s)", crdName, crdKind)
 	})
 
-	crNamespaces := map[string]string{}
+	return parseCrOutput(cmdOut)
+}
 
-	if cmdOut == "" {
+func parseCrOutput(rawOutput string) (map[string]string, error) {
+	crNamespaces := map[string]string{}
+	if rawOutput == "" {
 		// Filter out empty (0 CRs) output.
 		return crNamespaces, nil
 	}
 
-	lines := strings.Split(cmdOut, "\n")
+	lines := strings.Split(rawOutput, "\n")
 	for _, line := range lines {
 		lineFields := strings.Split(line, ",")
-		if len(lineFields) != expectedNumFields {
+		//nolint:gomnd
+		if len(lineFields) != 2 {
 			return crNamespaces, fmt.Errorf("failed to parse output line %s", line)
 		}
-		crNamespaces[lineFields[crNameFieldIdx]] = lineFields[namespaceFieldIdx]
+		// 0 = CR Name
+		// 1 = CR Namespace
+		crNamespaces[lineFields[0]] = lineFields[1]
```

Comment:
I'm a bit confused with this re-hardcoding of slice indexes/lenghts: aren't they magic numbers that should be avoided (as the linter suggests)?

Reply:
Yes, I agree. I brought the constants back.